packages/core/src/writers/split-tags-mode.ts

```diff
@@ -75,10 +75,7 @@ export const writeSplitTagsMode = async ({
         });
         mockData += builder.importsMock({
           implementation: implementationMock,
-          imports: [
-            ...importsForBuilder,
```

Comment:
So this `...importsForBuilder` was not necessary or not used?

Reply:
No, until now this was certainly necessary. For example, because I was using it to import response return types like `Pet` or `Pets`. With this change, they can now be covered by mock imports, so I removed them.